crates/server/src/datastore/postgres/mod.rs

```diff
@@ -757,4 +757,17 @@ impl DataStore for PostgresDataStore {
         package::PackageRecord::verify(key, record.content_bytes(), record.signature())
             .map_err(|_| DataStoreError::SignatureVerificationFailed)
     }
+
+    async fn debug_list_package_ids(&self) -> anyhow::Result<Vec<PackageId>> {
+        let mut conn = self.0.get().await?;
+        let names = schema::logs::table
+            .select(schema::logs::name)
+            .load::<Option<String>>(&mut conn)
```

Comment:
We can use `Text<PackageId>` here I think.

Reply:
Ah nice, will do.

Reply:
This ended up being a hassle because `PackageId`'s `FromStr:Err` doesn't match what `ParsedText` wants.

Reply:
Ah yeah, that makes sense. Can leave as-is then.